README-PRE-5.md

```diff
@@ -0,0 +1,374 @@
+# kubernetes-client
+
+[![Build Status][build]](https://travis-ci.org/godaddy/kubernetes-client) [![Greenkeeper badge][greenkeeper]](https://greenkeeper.io/)
+
+[greenkeeper]: https://badges.greenkeeper.io/godaddy/kubernetes-client.svg
+[build]: https://travis-ci.org/godaddy/kubernetes-client.svg?branch=master
+
+Simplified [Kubernetes API](http://kubernetes.io/) client for Node.js.
+
+This kubernetes-client API is deprecated. See [README.md](README.md)
+for the new API that has better support for all Kubernetes API
+endpoints.
+
+## Installation
+
+Install via npm:
+
+```console
+$ npm i kubernetes-client --save
+```
+
+## Examples
+
+kubernetes-client provides access to all the Kubernetes objects and
+includes some niceties for writing simpler code.
+
+### Basics
+
+kubernetes-client maps the URI paths in the Kubernetes API to
+sequences of objects chained together via properties and ending in a
+function. For example, to GET the `ReplicationController` named
+'http-rc' in the `Namespace` 'my-project':
+
+```js
+const Api = require('kubernetes-client');
+const core = new Api.Core({
+  url: 'http://my-k8s-api-server.com',
+  version: 'v1'  // Defaults to 'v1'
+});
+
+function print(err, result) {
+  console.log(JSON.stringify(err || result, null, 2));
+}
+
+core.namespaces('default').replicationcontrollers('http-rc').get(print);
```

Comment:
This uses the `default` namespace, but the description of the example on line 32 says it is going to use the `my-project` namespace - is this intentional?